lib/pbench/server/api/__init__.py

```diff
@@ -85,11 +261,12 @@ def get(self):
                     f":{app.config_server.get('pbench-receive-dir-prefix')}-002"
                 )
             )
-        except Exception:
+        except Exception as exc:
             app.logger.exception(
-                "There was something wrong constructing the host info."
+                "There was something wrong constructing the host info. %s",
+                exc.__class__,
             )
-            abort(500, message="There was something wrong with your request")
+            abort(400, message="There was something wrong with your request")
```

Comment:
Why this change from a `500` to a `400` error?  A `400` error code typically means the server is functioning fine, but encountered some problem with the request payload itself.  A `500` error typically means that the server failed to handle the request properly.

In this case `jsonify` should not raise any errors.  If it does, it indicates something is wrong with the server operation, not with their request payload.

Reply:
Ahh, make sense